src/BTCDepositAddressDeriver.sol

```diff
@@ -31,48 +44,71 @@ contract BTCDepositAddressDeriver {
         wasSeedSet = false;
     }
 
+    // Set Validators' joint pubkeys and network prefix, must be called after contract deployment
+    function setSeed(
+        string calldata _btcAddr1,
+        string calldata _btcAddr2,
+        BitcoinNetwork _network
+    ) public virtual {
+
+        string memory _hrp;
```

Comment:
nit: if remove this var and assign directly to `networkHrp`  we will reduce gas footprint?